tensorflow/python/kernel_tests/signal/fft_ops_test.py

```diff
@@ -554,6 +554,49 @@ def testGrad_Random(self):
             self._checkGradComplex(
                 self._tfIFFTForRank(rank), re, im, result_is_complex=False)
 
+class FFTShiftTest(test.TestCase):
+
+  def testDefinition(self):
```

Comment:
I think you need to decorate every method with `@test_util.run_deprecated_v1` as the above tests in this file are, otherwise the TF 2.0 tests fail.

Reply:
done :+1: